content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -315,6 +315,8 @@ tolerations to all daemons, to prevent DaemonSets from breaking.
 Adding these tolerations ensures backward compatibility. You can also add
 arbitrary tolerations to DaemonSets.
 
+> **Note:** If you manually schedule a pod by defining a node name in your workload yaml spec file like a pod, deployments, etc. it will override the usual scheduling mechanism, hence, bypassing the taint.
```

Comment:
```suggestion
{{< note >}}
If you manually schedule a pod by defining a node name in your workload yaml spec
file like a pod, deployments, etc. it will override the usual scheduling mechanism,
hence, bypassing the taint.
{{< /note >}}
```